### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/goreleaser ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1515443b13991dddc22ffa285ebc4d100e1aa9d8
  * Commits must be in the following format: **%{type}: %{description}**

 * Commit: 87fb11125b44d0443ca795a3e79f73d65383c121
  * Commits must be in the following format: **%{type}: %{description}**

 * Commit: 9f149c81513264169f8c5a305225c98727c3894c
  * Commits must be in the following format: **%{type}: %{description}**

Guidelines are available at https://github.com/goreleaser/goreleaser/blob/master/CONTRIBUTING.md#create-a-commit

---
This message was auto-generated by https://gitcop.com

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,3 +81,41 @@ See #284
...
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to goreleaser!
+<a href="graphs/contributors"><img src="https://opencollective.com/goreleaser/contributors.svg?width=890" /></a>
```
Comment (caarlos0) [inline CONTRIBUTING.md]: this URL doesn't seem to work...

Comment (monkeywithacupcake) [reply CONTRIBUTING.md]: bug submitted and actively being addressed at open collective. I'll check back here after status changes. Thanks!!!

Comment (caarlos0) [reply CONTRIBUTING.md]: Thanks =D

Comment (monkeywithacupcake) [reply CONTRIBUTING.md]: It works!

Comment (caarlos0) [reply CONTRIBUTING.md]: neat, thankkks!

**Comment by caarlos0** [comment]
hey @monkeywithacupcake , the sponsors part is broken now as well:

image

it seems intermittent actually, I have seen this before and it stopped happening before I reported, but now it's happening again...

**Pull request merged**